### DISCUSSION TITLE
App updates too frequent

### DISCUSSION THREAD

**Discussion by blogwebpl** [tutao/tutanota#6861] [Ideas]
- This is a feature request.

**Describe the bug**
Why are app updates so frequent? Almost every time I check my email, I'm asked if I want to install an update. It makes using Tuta unpleasant. There should be an option to disable updates if they are not critical.

**Comment by rezbyte** [comment]
Hi @blogwebpl,
Thank you for reaching out.

Please request features on the forum, via support or on the Discussions. This is so we can plan any potential improvements without cluttering our to-do list.
